cmd/vic-machine/create/create.go

```diff
@@ -1097,7 +1097,12 @@ func (c *Create) generateCertificates(server bool, client bool) ([]byte, *certif
 
 	var certs []byte
 	// generate the certs and keys with names conforming the default the docker client expects
-	err := os.MkdirAll(c.certPath, 0700)
+	files, err := ioutil.ReadDir(c.certPath)
+	if len(files) > 0 {
+		return nil, nil, fmt.Errorf("Specified directory to store certificates, \"%s\", is not empty. Please specify a new path in which to store generated certificates using --cert-path or remove the contents of \"%s\" and run vic-machine again.", c.certPath, c.certPath)
```

Comment:
@hickeng reviewing the code the only path I see, which is the one I was hitting, where we do not bail entirely upon failing to load certificates is when the `Leaf` of the certificate is `nil`. See the error on line 996 in `create.go`:

```log.Warnf("Failed to load x509 leaf: Unable to confirm server certificate cname matches provided cname %q. Continuing...", c.cname)```

I was hitting this path by using a wildcard certificate, which (apparently) has a `nil` `Leaf` in the certificate itself, presumably because it's not defined in a wildcard certificate due to the nature of that type of certificate. I don't think we want to bail in this situation, or we would block the usage of valid wildcard certificates, which is why this simply emits a warning.

Before my change, however, in that case the certificate would be overwritten on the filesystem if it existed in the same place as specified by `--cert-path`. Basically, this is a final-step cover-our-butts check for ridiculous corner cases like the one described (that I somehow managed to hit myself). The behavior you've requested is already present for the vast majority of cases.

Also I'm going to make that error statement more terse, as requested.